Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -29,6 +29,14 @@ public extension SCNVector3 {
     var length: SceneKitFloat {
         return sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
     }
+    
+    /// SwifterSwift: Returns the unit or normalized vector where `length = 1`.
+    ///
+    ///     SCNVector3(2, 3, 6).normalized  -> SCNVector3(2/7, 3/7, 6/7)
+    ///
+    var normalized: SCNVector3 {
+        SCNVector3(x / length, y / length, z / length)
```

Comment:
According to the project guidelines we can't reference other extensions. Also, doing it this way is inefficient as you're recalculating the length each time.
```suggestion
        let length = sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
        SCNVector3(x / length, y / length, z / length)
```